lib/lumberjack/client.rb

```diff
@@ -4,6 +4,7 @@
 require "thread"
 require "openssl"
 require "zlib"
+require "json"
```

Comment:
JRuby JSON is a bit slow and we don't use it inside Logstash we prefer to use @guyboertje's excellent jrjackson gem.
So we need a way to be able to override it without using multijson because of cost involved in using it.

we can keep can native json by default but we should be able to do something like this to use our own.
```
 Lumberjack::Client.json = LogStash::Json.method(:dump)
```